wallet/hardware.md

```diff
@@ -107,9 +107,9 @@ This type of set-up does come with additional hurdles that may be more difficult
 
 ***
 
-## Seed Signer
+## SeedSigner
 
-[Seed Signer](https://seedsigner.com/) is an offline, air-gapped Bitcoin transaction signing device that can be built from off-the-shelf [components](https://github.com/SeedSigner/seedsigner/blob/main/README.md#shopping-list) for around $50. After gathering and assembling the hardware, users flash the open source [software](https://github.com/SeedSigner/seedsigner/releases) onto a microSD card, insert to the device and power on. Seed Signer is a 'stateless' device, meaning that once powered off, it does not store any private keys. SeedSigner helps users save with Bitcoin by assisting with trustless private key generation and multi-signature wallet setup, and helps users transact with Bitcoin via a secure, air-gapped QR-exchange signing model.
+[SeedSigner](https://seedsigner.com/) is an offline, air-gapped Bitcoin transaction signing device that can be built from off-the-shelf [components](https://github.com/SeedSigner/seedsigner/blob/main/README.md#shopping-list) for around $70. After gathering and assembling the hardware, users flash the open source [software](https://github.com/SeedSigner/seedsigner/releases) onto a microSD card, insert to the device and power on. SeedSigner is a 'stateless' device, meaning that once powered off, it does not store any private keys. SeedSigner helps users save with Bitcoin by assisting with trustless private key generation and multi-signature wallet setup, and helps users transact with Bitcoin via a secure, air-gapped QR-exchange signing model.
 
 <p align="center">
 <img src="/assets/img/seedsigner(1).png" class=responsive width="410" height="300" maxheight="390">
```

Comment:
Need an image without background to suit the style of the page. 

I like to use a tool like https://www.remove.bg/

Reply:
will provide a better picture soon :+1: